cartoframes/vis/popup.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+
+from ..utils import gen_variable_name
+
+
+class Popup(object):
+    """Popup
+
+    Args:
+        data (dict): The popup definition for a layer. It contains the information
+          to show a popup on 'click' and 'hover' events with the attributes provided
+          in the definition using the `VL expressions syntax
```

Comment:
typo: using the CARTO VL expressions syntax (maybe is not a typo, but I wouldn't use only VL)